system/kube-monitoring/charts/prometheus-frontend/openstack-designate.alerts

```diff
@@ -14,3 +14,19 @@ ALERT OpenstackDesignateDnsBindDown
     summary = "DNS Bind server {{ $labels.kubernetes_name }} down.",
     description = "DNS Bind server {{ $labels.kubernetes_name }} down in region {{ $lables.region }}."
   }
+
+
+ALERT OpenstackDesignateZoneError
+  IF  (sum(sql_openstack_designate_zones{status="ERROR"}) by (host,status)) > 0
```

Comment:
You can save 2 characters by doing `sum(sql_openstack_designate_zones{status="ERROR"}) by (host,status) > 0`